meetings/2018-01-31.md

```diff
@@ -0,0 +1,255 @@
+       https://github.com/nodejs/abi-stable-node/issues/263#issuecomment-361711077
+      
+  * Modules
+    * dynamic import on by default for esm
+      * currently you need to use two flags to get dynamic import
+      * nodejs/node#18387
+  * import.meta implementation
+      * finding consensus around initial api
+      * nodejs/node#18368
+  * named exports for core modules
+      * Important feature we need to release without a flag
+      * Debate over if the exported modules should be filtered
+      * current implementation may create hazard for apms (@mcollina can speak to this)
+      * nodejs/node#18131 
+  * esm mode flag
+      * introduces semantic that could allow esm modules with the .js extension
+      * per package flags for loading mode
+      * still discussing implementation
+      * nodejs/node#18392
+
+* V8 currency
+  * 6.4 landed on master last week.
+  * 6.5 PR is open, no major issue.
+
+### nodejs/security-wg
+
+* Handle HackerOne org to core triage team [#103](https://github.com/nodejs/security-wg/issues/103)
+ * in last security-wg meeting, talking about using HackerOne for using modules, which is
+    working well. Not much input from core team yet.  More responses since then
+ * Vladimir requested and if he joined the core security triage team, it will make it easier
+   to do knowledge transfer
+ * Colin asked if there were any objections from those on the call to using HackerOne, none
+   voiced.
+
+## Q&A, Other
+* jdb, comments that HackerOne has a few bugs, some concern over how secure it is.  
+Suggestion is that he opens specific issues in the security-wg repo to raise the concerns.
+* gus, asking if https://github.com/nodejs/node/issues/18233 would address apm concerns. 
+* jdb, any plans to use N-API for ESM implementation.  Answer, long term thoughts to use
+  N-API for internal implementations but not there yet.
+
+## Upcoming Meetings
+
+* **Node.js Foundation Calendar**: https://nodejs.org/calendar
+
+Click `+GoogleCalendar` at the bottom right to add to your own Google calendar.
+# Node.js Foundation Technical Steering Committee (TSC) Meeting 2018-01-31
+
+## Links
+
+* **Recording**:  
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/478
+* **Minutes Google Doc**: https://docs.google.com/document/d/1R7Z9Rt01fqFddMnllJddezKwGIbaCcrkrGfw5Lhc_1A
+
+## Present
+
+* Ali Ijaz Sheikh @ofrobots (TSC)
+* Daniel Bevenius @danbev (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Evan Lucas @evanlucas (TSC)
+* Franziska Hinkelmann @fhinkel (TSC)
+* Jeremiah Senkpiel @Fishrock123 (TSC)
+* Michael Dawson @mhdawson (TSC)
+* MichaëZasso @targos (TSC)
+* Rich Trott @Trott (TSC)
+* Sakthipriyan Vairamani @thefourtheye (TSC)
+* Trevor Norris @trevnorris (TSC)
+* Matteo Collina @mcollina (TSC)
+
+## Agenda
+
+### Announcements
+ 
+*Extracted from **tsc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
```

Comment:
I'll update it in this PR, but can you update the template to use `labeled` (preferred US English spelling)?